Scan/CK3/pinout

```diff
@@ -0,0 +1,88 @@
+Pin Usage
```

Comment:
This file isn't necessary. I just include it with Input Club's keyboards as a reference to how the schematic is laid out.